app/task_routes.py

```diff
@@ -0,0 +1,122 @@
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, make_response, jsonify, abort
+import datetime
+import os
+import requests
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+# create helper function to post message to Slack
+def slack_bot(slack_message):
```

Comment:
Consider renaming this method to include a verb to clarify what this method does, maybe like post_to_slack()